### PR TITLE
[type:fix] configs import and export refactor

### LINKED ISSUE #5826: [BUG] ID conflict during import config
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

_No response_

### Expected Behavior

_No response_

### Steps To Reproduce

_No response_

### Environment

```markdown
ShenYu version(s):
```

### Debug logs

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by eye-gu**
close #5826 

Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `./mvnw clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/service/configs/SelectorDataConfigsExportImportHandler.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        ConfigImportResult configImportResult = selectorService.importData(namespaceId, selectorList, context);
+        context.getResult().put(ExportImportConstants.SELECTOR_IMPORT_SUCCESS_COUNT, configImportResult.getSuccessCount());
+        if (StringUtils.isNotEmpty(configImportResult.getFailMessage())) {
+            context.getResult().put(ExportImportConstants.SELECTOR_IMPORT_FAIL_MESSAGE, configImportResult.getFailMessage());
+        }
+    }
+
+    @Override
+    public int importOrder() {
+        return 3;
```
Comment (moremind) [inline shenyu-admin/src/main/java/org/apache/shenyu/admin/service/configs/SelectorDataConfigsExportImportHandler.java:70]: refactor to enum

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/service/configs/ConfigsExportImportEnum.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+    /**
+     * rule.
+     */
+    Rule("rule.json", 4),
+
+    /**
+     * dict.
+     */
+    Dict("dict.json", 5),
```
Comment (moremind) [inline shenyu-admin/src/main/java/org/apache/shenyu/admin/service/configs/ConfigsExportImportEnum.java:50]: may include dict and plugin handler?

**Review by moremind — CHANGES_REQUESTED** [review]

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/service/configs/ConfigsExportImportEnum.java (hunk 2 above)
Comment (eye-gu) [reply shenyu-admin/src/main/java/org/apache/shenyu/admin/service/configs/ConfigsExportImportEnum.java:50]: Currently, it includes both of these

Comment (moremind) [reply shenyu-admin/src/main/java/org/apache/shenyu/admin/service/configs/ConfigsExportImportEnum.java:50]: I think you should add plugin handler

**Review by moremind — CHANGES_REQUESTED** [review]

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/service/configs/ConfigsExportImportEnum.java (hunk 2 above)
Comment (eye-gu) [reply shenyu-admin/src/main/java/org/apache/shenyu/admin/service/configs/ConfigsExportImportEnum.java:50]: The plugin handler is associated with the plugin

**Review by moremind — APPROVED** [review]

**Pull request merged**